packages/resourceful-action-creators/test/unit/resourceful-action-creators.js

```diff
@@ -473,7 +473,7 @@ describe('Resourceful Action Creators', function() {
             url: 'http://www.google.com',
             method: 'GET',
           }
-        });
+        }, undefined);
```

Comment:
The tests actually break because of `calledWithExactly` - the code is calling `foo(bar, baz)` and that was breaking the tests because it was actually being called as `foo(bar, baz, undefined)` when a `callback` was not passed in.